doc/source/cluster/kubernetes/examples/verl-post-training.md

```diff
@@ -0,0 +1,150 @@
+(kuberay-verl)=
+# Reinforcement Learning with Human Feedback (RLHF) for LLMs with verl on KubeRay
+
+[verl](https://github.com/volcengine/verl) is an open-source framework that provides a flexible, efficient, and production-ready RL training library for large language models (LLMs).
+This guide demonstrates PPO training on the GSM8K dataset with verl for `Qwen2.5-0.5B-Instruct` on KubeRay.
+
+* To make it easier to follow, this guide launches a single-node RayCluster with 4 GPUs.
+You can easily use KubeRay to launch a multi-node RayCluster to train larger models.
+* You can also use the [RayJob CRD](kuberay-rayjob-quickstart) for production use cases.
+
+# Step 1: Create a Kubernetes cluster with GPUs
+
+Follow the instructions in [Managed Kubernetes services](kuberay-k8s-setup) to create a Kubernetes cluster with GPUs.
+
+This guide uses a Kubernetes cluster with 4 L4 GPUs.
+
+For GKE, you can follow the instructions in [this tutorial](kuberay-gke-gpu-cluster-setup) and use the following command
+to create a GPU node pool with 4 L4 GPUs per Kubernetes node:
+
+```bash
+gcloud container node-pools create gpu-node-pool \
+  --accelerator type=nvidia-l4-vws,count=4 \
+  --zone us-west1-b \
+  --cluster kuberay-gpu-cluster \
+  --num-nodes 1 \
+  --min-nodes 0 \
+  --max-nodes 1 \
+  --enable-autoscaling \
+  --machine-type g2-standard-48
+```
+
+# Step 2: Install KubeRay operator
+
+Follow the instructions in [KubeRay operator](kuberay-operator-deploy) to install the KubeRay operator.
+
+# Step 3: Create a RayCluster
+
+```sh
+kubectl apply -f https://raw.githubusercontent.com/ray-project/kuberay/master/ray-operator/config/samples/ray-cluster.verl.yaml
+```
+
+# Step 4: Install verl in the head Pod
+
+Log in to the head Pod and install verl.
+This step won't be necessary when the verl community supports images with verl pre-installed in the future.
```

Comment:
```suggestion
The verl community does not support building images with verl installed.
```

I understand the sentiment here, but try to avoid future state. Wondering if there is an open GitHub issue requesting this functionality you can link to? (Or can we open one?) You could then add something like "Track the request for this functionality [here](#link)."

Reply:
updated